nixos/modules/services/monitoring/netdata.nix

```diff
@@ -158,11 +148,31 @@ in {
         # User and group
         User = cfg.user;
         Group = cfg.group;
-        # Runtime directory and mode
-        RuntimeDirectory = "netdata";
-        RuntimeDirectoryMode = "0755";
         # Performance
         LimitNOFILE = "30000";
+        # Runtime directory and mode
```

Comment:
I would say none of these options depend on a particular systemd version. Systemd will just ignore unknown options. The only side-effect is that if options like `RuntimeDirectory` are not available the same directories have to be created through other means i.e. packaging scripts/package manager.

Reply:
Need to fix something?

Reply:
No. this was just for the netdata maintainers.